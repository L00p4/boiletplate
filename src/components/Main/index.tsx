import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Logo src="/img/logo.svg" alt="" />
    <S.Title>React avan√ßado</S.Title>
    <S.Description>
      TypeScript, ReactJS, NextJS, e Styled Components
    </S.Description>

    <S.Illustration src="/img/computerGuy.svg" alt="" />
  </S.Wrapper>
)

export default Main
