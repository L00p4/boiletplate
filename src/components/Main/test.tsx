import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

import Main from '.'

describe('<Main />', () => {
  it('shoud render heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
