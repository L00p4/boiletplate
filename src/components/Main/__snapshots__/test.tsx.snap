// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> shoud render heading 1`] = `
<main>
  <h2>
    React avan√ßado
  </h2>
</main>
`
